src/literals.py

```diff
@@ -148,3 +153,11 @@ class Status(Enum):
         "version": "3.6.1",
     },
 }
+
+
+class Role(str, Enum):
+    BROKER = "broker"
+    PARTITIONER = "partitioner"
+
+
+SERVICES = {Role.BROKER: "daemon", Role.PARTITIONER: "cruise-control"}
```

Comment:
**thought/suggestion:** Maybe we can make a more complex 'role' type, with attrs associating to this stuff, and extra metadata for service etc. Maybe PATHS too?

Reply:
I gave it a try in 07acec4d0df854b14313445566757c0f82555183, I like it. We loose the exhaustiveness of match + enum, but I think the pros of this approach outnumber the cons